akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -279,7 +279,7 @@ object FlowGraph {
    */
   def builder[M](): Builder[M] = new Builder()(new scaladsl.FlowGraph.Builder[M])
 
-  final class Builder[Mat]()(private implicit val delegate: scaladsl.FlowGraph.Builder[Mat]) { self ⇒
+  final class Builder[+Mat]()(private implicit val delegate: scaladsl.FlowGraph.Builder[Mat]) { self ⇒
```

Comment:
added this `+` because of "covariant type M occurs in invariant position in type ⇒ akka.stream.javadsl.FlowGraph.Builder[M] of method asJava"